src/Interpreters/ExpressionAnalyzer.cpp

```diff
@@ -174,28 +174,38 @@ void tryTranslateToParametricAggregateFunction(
     }
     else if (lower_name.starts_with("quantile") && !lower_name.ends_with("if"))
```

Comment:
so far, we need block all aggregate functions with suffix `AggregateFunctionCombinatorFactory::tryFindSuffix`